test/PositionManager.ts

```diff
@@ -123,18 +171,167 @@ describe("GammaPoolFactory", function () {
+            
+            const { args } = res.events[1];
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.assets.toNumber()).to.equal(16);
+        });
+
+        it("#depositReserves should return shares and length of reserves", async function () {
+            await cfmm.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+            const DepositReservesParams =  {
+                cfmm: cfmm.address,
+                amountsDesired: [10000, 100],
+                amountsMin: [1000, 10],
+                to: addr4.address,
+                protocol: 1,
+                deadline: ethers.constants.MaxUint256
+            }
+            const res = await (await posMgr.depositReserves(DepositReservesParams)).wait();
+            
+            const { args } = res.events[0];
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.reservesLen).to.equal(4);
+            expect(args.shares.toNumber()).to.equal(18);
+        });
+
+        it("#withdrawReserves should return assets and lenght of reserves", async function () {            
+            const WithdrawReservesParams =  {
+                cfmm: cfmm.address,
+                protocol: 1,
+                amount: 1000,
+                amountsMin: [100, 200, 300],
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+
+            const res = await (await posMgr.withdrawReserves(WithdrawReservesParams)).wait();
+
+            const { args } = res.events[1];
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.reservesLen).to.equal(3);
+            expect(args.assets.toNumber()).to.equal(17);
+        });
     });
+
     // You can nest describe calls to create subsections.
-    describe("Long Gamma", function () {
+    describe("Long Gamma Functions", function () {
+        it("#createLoan should return tokenId", async function () {
+            expect(tokenId.toNumber()).to.equal(1);
+        });
+
+        it("#borrowLiquidity should return tokenId", async function () {
+            const BorrowLiquidityParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
+                lpTokens: 1,
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+            
+            const res = await (await posMgr.borrowLiquidity(BorrowLiquidityParams)).wait();
+            
+            const { args } = res.events[0]
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.tokenId.toNumber()).to.equal(1);
+        });
+
+        it("#repayLiquidity should return tokenId, paid liquidity, paid lp tokens and length of amounts array", async function () {
+            const RepayLiquidityParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
```

Comment:
use tokenId from beforeEach here, don't just use 1. So it should be "tokenId: tokenId,"